lib/oli/accounts/schemas/author.ex

```diff
@@ -91,6 +91,23 @@ defmodule Oli.Accounts.Author do
     |> maybe_name_from_given_and_family()
   end
 
+  @doc """
+  Creates a changeset that is used to update an author's profile
+  """
+
+  def update_author_changeset(author, attrs \\ %{}) do
+    author
+    |> pow_changeset(attrs)
+    |> cast(attrs, [
+      :given_name,
+      :family_name,
+      :email
```

Comment:
What do you think setting this four lines just in one?

Same for the User side.